packages/components/watermark/src/useClips.ts

```diff
@@ -148,7 +156,7 @@ export default function useClips() {
         cutWidth,
         cutHeight,
         targetX,
-        targetY,
+        targetY - baselineOffset,
```

Comment:
```suggestion
        targetY + baselineOffset,
```

It should be offset downward here.

Before

https://github.com/user-attachments/assets/84fee378-f096-476b-9e48-1287f205b3de

After

https://github.com/user-attachments/assets/3447b029-5e71-44ee-b9e9-d5691bc6ba47